subprojects/testing-jvm/src/main/java/org/gradle/api/tasks/testing/Test.java

```diff
@@ -1165,7 +1152,6 @@ void setTestExecuter(TestExecuter<JvmTestExecutionSpec> testExecuter) {
      *
      * @since 6.7
      */
-    @Incubating
```

Comment:
This method was already added to the list in the release notes :+1:

Reply:
Thanks